x-pack/plugins/security_solution/public/common/components/centered_loading_spinner/centered_loading_spinner.tsx

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { useMemo } from 'react';
+import { EuiLoadingSpinner, useEuiTheme, type EuiLoadingSpinnerProps } from '@elastic/eui';
+
+interface CenteredLoadingSpinnerProps extends EuiLoadingSpinnerProps {
+  topOffset?: string;
+}
+
+export const CenteredLoadingSpinner = React.memo<CenteredLoadingSpinnerProps>(
```

Comment:
nit: for my information isnt

```tsx
interface Props {
}

const ComponentNameComponent: FC<Props>() => {
    return ...
}

ComponentNameComponent.displayName = 'ComponentNameComponent'

export const ComponentName = memo(ComponentName)
```

the accepted style to writing the components here? Or is it specific to ecs package only?

Reply:
That's Andrew's style 😉
I prefer this style, it's easier to find the component's implementation and instances with the IDE, and it does not require the extra line to memo and export. It's pretty common in Security and Kibana.

Reply:
That's a very good point Sergi I didn't look at it that way. Thanks.